### PR TITLE
[15.0][MIG] tms: Migration to 15.0 

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT37730

### CODE REVIEW

**Comment by cesar-tecnativa** [comment]
@sergio-teruel please review

**Comment by cesar-tecnativa** [comment]
@CarlosRoca13 please review

File: tms/models/project_task.py [hunk 1]
```diff
@@ -112,14 +112,8 @@ class ProjectTask(models.Model):
         },
     )
     unload_service = fields.Boolean(string="Unload")
-    distance_estimated = fields.Float(
-        digits="TMS Distance",
```
Comment (CarlosRoca13) [inline tms/models/project_task.py]: Why remove this?

File: tms/models/project_task.py [hunk 2]
```diff
@@ -112,14 +112,8 @@ class ProjectTask(models.Model):
...
-        string="Distance Estimated",
-    )
-    distance_traveled = fields.Float(
-        digits="TMS Distance",
```
Comment (CarlosRoca13) [inline tms/models/project_task.py]: Same

File: tms/views/assets.xml [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-        <link rel="stylesheet" href="/tms/static/src/css/kanban_view.scss" />
-    </xpath>
-</template>
+    <template id="assets_backend" name="tms assets" inherit_id="web.assets_backend">
+        <xpath expr="." position="inside">
+            <link rel="stylesheet" href="/tms/static/src/css/kanban_view.scss" />
+        </xpath>
+    </template>
 
 </odoo>
```
Comment (CarlosRoca13) [inline tms/views/assets.xml]: This file should be declared at manifest 

See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0 8th point

**Review by CarlosRoca13 — COMMENTED** [review]
Some comments

**Comment by pedrobaeza** [comment]
Haz rebase

**Comment by cesar-tecnativa** [comment]
> Haz rebase

done

**Comment by pedrobaeza** [comment]
Está en rojo aún. Asegúrate que está en verde antes de pedir revisiones.

**Comment by cesar-tecnativa** [comment]
Waiting for https://github.com/OCA/sale-workflow/pull/2108

**Pull request merged**